src/test/java/org/snomed/aag/rest/ServiceIntegrationControllerTest.java

```diff
@@ -244,6 +252,33 @@ void receiveCommitInformation_ShouldReturnExpectedResponse_WhenRequestingTwice()
 		assertResponseStatus(secondRequest, 409); // Project has been incremented and PAC hasn't been completed for that iteration.
 	}
 
+	@Test
+	void receiveCommitInformation_ShouldReturnExpectedResponse_WhenMetadataOverwritesPAC() throws Exception {
+		// given
+		String requestUrl = receiveCommitInformation();
+		String projectPath = "MAIN/projectA";
+		String taskPath = "MAIN/projectA/taskB";
+		CommitInformation commitInformation = new CommitInformation(taskPath, CommitInformation.CommitType.PROMOTION, 1L, Collections.emptyMap());
+		String projectCriteriaId = "project-criteria-id";
+		String taskCriteriaId = "task-criteria-id";
+
+		givenProjectAcceptanceCriteriaExists(projectPath, 1, projectCriteriaId, taskCriteriaId);
+		givenCriteriaItemExists(projectCriteriaId, true, 0, projectCriteriaId, AuthoringLevel.PROJECT, "complex");
+		givenCriteriaItemExists(taskCriteriaId, true, 1, taskCriteriaId, AuthoringLevel.TASK, "complex");
+		givenBranchDoesExist(taskPath, buildMetadataWithAuthorFlag(new LinkedHashMap<>(Map.of("complex", false))));
```

Comment:
`Map.of()` produces an unmodifiable collection... I had to nest it within `new LinkedHashMap<>()`